content/content-strategy/define-user-content-needs/index.yml

```diff
@@ -0,0 +1,17 @@
+pagetitle: Define user content needs
+weight: 10
```

Comment:
Check value against all other sibling, child and parent page `index.yml` pages. Will affect list order if these are all the same value.

Reply:
Checked